features/step_definitions/form_steps.rb

```diff
@@ -86,12 +86,13 @@
 end
 
 When "(I )(they )select {string} from {string}" do |value, field|
-  if page.has_css?('.pf-c-form__label', text: field, wait: 0)
```

Comment:
This didn't work for this case, because `.pf-c-form__label` is added also to non-React forms where the selects are `<select>` element (not `<input>` like in React)

Reply:
React has both input-selects and select-selects so before removing this line I would fix the input builder (or whatever the responsible code) to prevent adding `pf-c-form__label` to non-react forms.